### PR TITLE
Release/0.11.0

### PR DESCRIPTION
**Pull request by dilyand**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,20 @@ These events can then be viewed in Kibana at `http://< sp mini public ip>/kibana
...
+### `The box 'ubuntu/xenial64' could not be found or could not be accessed in the remote catalog.`
+
+Your Vagrant version is probably outdated. Use Vagrant 2.0.0+.
+
+### `npm install` results in `enoent ENOENT: no such file or directory, open '/package.json'`
+
+This is caused by trying to use NFS. Comment the relevant lines in `Vagrantfile`.
+
+Most likely this will happen on `TASK [sp_mini_5_build_ui : Install npm packages based on package.json.]` but see also: https://discourse.snowplowanalytics.com/t/snowplow-mini-local-vagrant/2930.
+
```
Comment (lukeindykiewicz) [inline README.md:71]: Awesome you added some q/a!

**Review by lukeindykiewicz — APPROVED** [review]
I'm not an expert on Mini, but lgtm :)

File: provisioning/resources/ui/js/components/Overview.tsx [hunk 2]
```diff
@@ -42,12 +42,15 @@ export class Overview extends React.Component<{}, {}> {
         <h3>3. Understanding how Snowplow Mini works</h3>
         <h3>Quicklinks: </h3>
         <ul>
+          <li><a href={'https://docs.snowplowanalytics.com/docs/understanding-your-pipeline/what-is-snowplow-mini'}>What is Snowplow Mini?</a></li>
+          <li><a href={'https://docs.snowplowanalytics.com/docs/open-source-components-and-applications/snowplow-mini/snowplow-mini-0-10-1/usage-guide/'}>Usage guide</a></li>
+          <li><a href={'https://docs.snowplowanalytics.com/docs/open-source-components-and-applications/snowplow-mini/snowplow-mini-0-10-1/control-plane-api/'}>Control Plane API</a></li>
```
Comment (oguzhanunlu) [inline provisioning/resources/ui/js/components/Overview.tsx:47]: It points to 0.10.1 docs but I assume we'll have new doc pages dedicated to 0.11.0

Comment (dilyand) [reply provisioning/resources/ui/js/components/Overview.tsx]: Fixed.

Comment (oguzhanunlu) [reply provisioning/resources/ui/js/components/Overview.tsx]: thanks!

**Comment by dilyand** [comment]
AWS:

```
| Ⰱ :> curl -i -X POST http://aws-sandbox-dilyan.mini.snplow.net/r/tp2 -d '{}'
HTTP/1.1 404 Not Found
Date: Mon, 26 Oct 2020 08:28:13 GMT
Content-Type: text/plain; charset=UTF-8
Content-Length: 18
Connection: keep-alive
Server: Caddy
Server: akka-http/10.1.10

redirects disabled
```

***

```
| Ⰱ :> curl -i -X POST http://aws-sandbox-dilyan.mini.snplow.net/com.snowplowanalytics.snowplow/tp2 -H 'Authorization: Basic REDACTED' -d '{}'
HTTP/1.1 200 OK
Date: Mon, 26 Oct 2020 08:59:11 GMT
Content-Type: text/plain; charset=UTF-8
Content-Length: 2
Connection: keep-alive
Access-Control-Allow-Credentials: true
Access-Control-Allow-Origin: *
P3p: policyref="/w3c/p3p.xml", CP="NOI DSP COR NID PSA OUR IND COM NAV STA"
Server: Caddy
Server: akka-http/10.1.10
Set-Cookie: sp=9c37e5e2-c185-4d5b-821a-6567699b7515; Expires=Tue, 26 Oct 2021 08:59:11 GMT; Path=/; Secure; SameSite=None

ok
```

> 

data.payload.headers |   | Timeout-Access: <function1>, Host: localhost:8080, User-Agent: curl/7.54.0, Accept: */*, X-Amzn-Trace-Id: Root=1-5f968fdd-26b5889f05f731980caaef71, X-Forwarded-For: 18.194.133.57, 10.100.2.221, X-Forwarded-Port: 80, X-Forwarded-Proto: http, Accept-Encoding: gzip, application/x-www-form-urlencoded
-- | -- | --

**Comment by dilyand** [comment]
`0.11.0-rc1` uses Enrich `1.4.0` not `1.4.1`.

**Pull request closed without merging**